seatunnel-core/seatunnel-starter/src/main/java/org/apache/seatunnel/core/starter/seatunnel/command/ClientExecuteCommand.java

```diff
@@ -112,14 +114,18 @@ public void execute() throws CommandExecuteException {
                 startTime = LocalDateTime.now();
                 // create job proxy
                 ClientJobProxy clientJobProxy = jobExecutionEnv.execute();
+                // register cancelJob hook
+                if (clientCommandArgs.isCloseJob()) {
+                    Runtime.getRuntime().addShutdownHook(new Thread(() -> shutdownHook(clientJobProxy)));
```

Comment:
I'm not sure if shutdownHook throw some exception, the console can show it? If not, please use try catch then log it.